test/tempfile_test.py

```diff
@@ -0,0 +1,95 @@
+import os
+import time
+import pytest
+import ray
+import ray.tempfile_services as tempfile_services
+
+
+def test_conn_cluster():
+    try:
+        ray.init(
+            redis_address="127.0.0.1:6379", temp_dir="/tmp/this_should_fail")
```

Comment:
use this way to assert exceptions: https://stackoverflow.com/questions/23337471/how-to-properly-assert-that-an-exception-gets-raised-in-pytest